helm/settings_local.py

```diff
@@ -1,124 +1,231 @@
 # Copyright The IETF Trust 2007-2024, All Rights Reserved
 # -*- coding: utf-8 -*-
 
+from base64 import b64decode
+from email.utils import parseaddr
+
+from ietf import __release_hash__
 from ietf.settings import *                                          # pyflakes:ignore
 
-ALLOWED_HOSTS = ['*']
+USE_TZ = True
+
+# Secrets
+
+SECRET_KEY = os.environ.get("DATATRACKER_DJANGO_SECRET_KEY")
```

Comment:
One possibility is to wrap the `os.environ.get` calls here so that if the value is "file:/foo/bar/baz" you make the value returned read the specified file.

Reply:
Interesting thought. At the moment the environment is the method we expect to be most convenient for production deployment, but something like this might help if/when we try to move back to a more unified setup. Will keep it in mind. Thanks!